platform/darwin/darwin.xcconfig

```diff
@@ -56,7 +56,19 @@ GCC_WARN_UNUSED_FUNCTION = YES
 GCC_WARN_UNUSED_LABEL = YES
 GCC_WARN_UNUSED_VARIABLE = YES
 
-HEADER_SEARCH_PATHS = $(inherited) "$(PROJECT_DIR)/../../vendor/mapbox-gl-native/include" "$(PROJECT_DIR)/../../vendor/mapbox-gl-native/platform/default/include" "$(PROJECT_DIR)/../../vendor/mapbox-gl-native/vendor/mapbox-base/mapbox/variant/include" "$(PROJECT_DIR)/../../vendor/mapbox-gl-native/vendor/mapbox-base/mapbox/geometry.hpp/include" "$(PROJECT_DIR)/../../vendor/mapbox-gl-native/vendor/mapbox-base/mapbox/value/include" "$(PROJECT_DIR)/../../vendor/mapbox-gl-native/vendor/mapbox-base/mapbox/geojson.hpp/include" "$(PROJECT_DIR)/../../vendor/mapbox-gl-native/vendor/mapbox-base/mapbox/optional" "$(PROJECT_DIR)/../../vendor/mapbox-gl-native/vendor/mapbox-base/mapbox/typewrapper/include" "$(PROJECT_DIR)/../../vendor/mapbox-gl-native/vendor/mapbox-base/mapbox/weak/include" "$(PROJECT_DIR)/../../vendor/mapbox-gl-native/vendor/mapbox-base/extras/rapidjson/include" "$(PROJECT_DIR)/../../vendor/mapbox-gl-native/vendor/mapbox-base/extras/expected-lite/include" "$(PROJECT_DIR)/../../vendor/mapbox-gl-native/vendor/polylabel/include" "$(PROJECT_DIR)/../../platform/darwin/include" "$(PROJECT_DIR)/../../vendor/mapbox-gl-native/platform/darwin/include"
+HEADER_SEARCH_PATHS = $(inherited)
+HEADER_SEARCH_PATHS = $(HEADER_SEARCH_PATHS) "$(PROJECT_DIR)/../../vendor/mapbox-gl-native/vendor/mapbox-base/include"
+HEADER_SEARCH_PATHS = $(HEADER_SEARCH_PATHS) "$(PROJECT_DIR)/../../vendor/mapbox-gl-native/vendor/mapbox-base/mapbox/geojson.hpp/include"
+HEADER_SEARCH_PATHS = $(HEADER_SEARCH_PATHS) "$(PROJECT_DIR)/../../vendor/mapbox-gl-native/vendor/mapbox-base/mapbox/geometry.hpp/include"
+HEADER_SEARCH_PATHS = $(HEADER_SEARCH_PATHS) "$(PROJECT_DIR)/../../vendor/mapbox-gl-native/vendor/mapbox-base/mapbox/optional"
+HEADER_SEARCH_PATHS = $(HEADER_SEARCH_PATHS) "$(PROJECT_DIR)/../../vendor/mapbox-gl-native/vendor/mapbox-base/mapbox/variant/include"
+HEADER_SEARCH_PATHS = $(HEADER_SEARCH_PATHS) "$(PROJECT_DIR)/../../vendor/mapbox-gl-native/vendor/mapbox-base/extras/expected-lite/include"
+HEADER_SEARCH_PATHS = $(HEADER_SEARCH_PATHS) "$(PROJECT_DIR)/../../vendor/mapbox-gl-native/vendor/mapbox-base/extras/rapidjson/include"
+HEADER_SEARCH_PATHS = $(HEADER_SEARCH_PATHS) "$(PROJECT_DIR)/../../vendor/mapbox-gl-native/vendor/polylabel/include"
+HEADER_SEARCH_PATHS = $(HEADER_SEARCH_PATHS) "$(PROJECT_DIR)/../../platform/darwin/include"
+HEADER_SEARCH_PATHS = $(HEADER_SEARCH_PATHS) "$(PROJECT_DIR)/../../vendor/mapbox-gl-native/include"
+HEADER_SEARCH_PATHS = $(HEADER_SEARCH_PATHS) "$(PROJECT_DIR)/../../vendor/mapbox-gl-native/platform/default/include"
+HEADER_SEARCH_PATHS = $(HEADER_SEARCH_PATHS) "$(PROJECT_DIR)/../../vendor/mapbox-gl-native/platform/darwin/include"
```

Comment:
I think .xcconfig is more like a property list than a script. If you specify `HEADER_SEARCH_PATHS` multiple times, each new specification overrides the previous one. I think that’s why the builds are all failing with stuff like:

```
❌  /Users/distiller/project/vendor/mapbox-gl-native/include/mbgl/util/geometry.hpp:3:10: 'mapbox/geometry/geometry.hpp' file not found



#include <mapbox/geometry/geometry.hpp>

         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
This is already done.